support/cov-combine.js

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env node
+/*
+ * Combines multiple istanbul coverage JSON result files to create one
+ * coverage HTML that covers all the individual results.
+ *
+ * E.g.
+ * $ cov-combine.js cov-pt[0-9]/coverage.json
```

Comment:
BTW, I wanted to write

```sh
cov-combine.js cov-pt*/coverage.json
```

here but then JS recognized `*/` as end of comment block so...